package.json

```diff
@@ -5,13 +5,12 @@
   "description": "React components for Wonder Blocks design system.",
   "main": "index.js",
   "engines": {
-    "node": ">=12.22"
+    "node": ">=16"
   },
   "scripts": {
     "alex": "alex packages/",
-    "build:all": "yarn run build:cjs && yarn run build:es6",
-    "build:cjs": "webpack --config ./build-settings/webpack.config.js --display-modules && node ./utils/package-flow-types.js",
-    "build:es6": "rollup -c ./build-settings/rollup.config.js",
+    "build": "rollup -c ./build-settings/rollup.config.js",
+    "build:es6": "yarn run build",
```

Comment:
**note:** I kept this command just to make the `Check build sizes` workflow step happy. I'll remove it later in an Audit.